src/InflectorConfig.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace TomPHP\ConfigServiceProvider;
+
+use ArrayIterator;
+use IteratorAggregate;
+
+final class InflectorConfig implements IteratorAggregate
+{
+    /** @var array */
+    private $inflectors;
+
+    /**
+     * @param array $config
+     */
```

Comment:
@localheinz how do you feel about only have `@param` docs for arguments which aren't typed in the method signature? Normally I'd have something like this:

```php
/**
 * @param string $name
 *
 * @return bool
 */
public function doSomething($name, Config $config) { /* ... */ }
```